### DISCUSSION TITLE
Is there a way to turn the render log off when previewing html formats?

### DISCUSSION THREAD

**Discussion by juliantao** [quarto-dev/quarto-cli#6436] [Q&A]
### Description

I noticed a sub-window showing the render logs in the HTML preview window, but I can't recall when it started appearing. While I understand that this feature aims to help monitor the rendering process and facilitate debugging, I personally find it quite distracting.

Is there any way to disable or turn off this sub-window for render logs?

Thank you very much!

**Comment by mcanouil** [comment]
I must confess, I have no clue what you are talking about.
What window? Which IDE? What log are you talking about? Is it log or the progress messages?

Did you activate any advanced log options? (https://quarto.org/docs/troubleshooting/index.html)

**Comment by juliantao** [comment]
Sorry that I did not make it clear.  I was probably using the wrong term. Check the short vid below.


https://github.com/quarto-dev/quarto-cli/assets/60116296/de703b6a-8fab-471c-b991-1b9e831cf06c

**Reply by mcanouil** [reply]
You still did not answer and give details.
There is no reason for the HTML to be displayed if there are errors in your Python code cell.
It's not log but actual compiling done by the VSCode extension (I am guessing), thus not related to Quarto CLI.
The "log" is removed when compiling properly, so I don't see the issue.
See <https://github.com/quarto-dev/quarto>.
